client/errors.go

```diff
@@ -0,0 +1,101 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package client
+
+// XXX: sync with the error kinds in daemon and ensure we define them
+//      only in a single place
+const (
+	// ErrorKindTwoFactorRequired the client needs to retry the login command including an OTP
+	ErrorKindTwoFactorRequired = "two-factor-required"
+	// ErrorKindTwoFactorFailed: the OTP provided wasn’t recognised
+	ErrorKindTwoFactorFailed = "two-factor-failed"
+	// ErrorKindLoginRequired: the requested operation cannot be performed without an authenticated user. This is the kind of any other 401 Unauthorized response.
+	ErrorKindLoginRequired = "login-required"
+	// ErrorKindInvalidAuthData: the authentication data provided failed to validate (e.g. a malformed email address). The value of the error is an object with a key per failed field and a list of the failures on each field.
+	ErrorKindInvalidAuthData = "invalid-auth-data"
+	// ErrorKindTermsNotAccepted: deprecated, do not document
+	ErrorKindTermsNotAccepted = "terms-not-accepted"
+	// ErrorKindNoPaymentMethods: deprecated, do not document
+	ErrorKindNoPaymentMethods = "no-payment-methods"
+	// ErrorKindPaymentDeclined: deprecated, do not document
+	ErrorKindPaymentDeclined = "payment-declined"
+	// ErrorKindPasswordPolicy: provided password doesn’t meet system policy.	ErrorKindPasswordPolicy    = "password-policy"
```

Comment:
Something happened here.
```suggestion
	// ErrorKindPasswordPolicy: provided password doesn’t meet system policy.	
	ErrorKindPasswordPolicy    = "password-policy"
```

Reply:
Uh, thanks! fixed and pushed.